src/components/CommandBar/CommandBarSelectionInput.tsx

```diff
@@ -112,6 +112,23 @@ function CommandBarSelectionInput({
     onSubmit(selection)
   }
 
+  // Clear selection if needed
+  useEffect(() => {
+    arg.clearSelectionFirst &&
+      engineCommandManager.modelingSend({
+        type: 'Set selection',
+        data: {
+          selectionType: 'singleCodeCursor',
+        },
+      })
+  }, [arg.clearSelectionFirst])
```

Comment:
If there's two scene selections inputs back to back as is the case for subtract, I think it makes sense to clear the user's selection between yeah?

Maybe in future we can change the colour of the old selection, but that would require engine endpoints or similar, which I don't think is worth it atm.